.travis.yml

```diff
@@ -8,6 +8,7 @@ cache:
 branches:
   only:
     - master
+    - fix/semver-support
```

Comment:
I think this has to be removed. See https://github.com/Carthage/Carthage/pull/2392#issuecomment-377610046

Reply:
@blender can you remove this line? I can't edit it